app/controllers/application_controller.rb

```diff
@@ -43,14 +43,19 @@ def top_level_error_handler(exception = nil)
   end
 
   def after_sign_in_path_for(_resource)
-    target = params.fetch(:previous_url, user_root_path)
-    if target.start_with?("/account") || target.start_with?("/oauth")
+    target = params[:previous_url]
```

Comment:
We're losing the defaul here, is there a better one if there is no previous URL?

Reply:
It was kind of redundant anyway, as the `else` cause goes to `user_root_path`